deafrica/data/chirps.py

```diff
@@ -29,6 +30,11 @@
 log.info("Starting CHIRPS downloader")
 
 
+def check_for_compression(href):
```

Comment:
This doesn't check for compression, it checks the URL is valid

Reply:
Updated the function name. I think the new name is clearer